scripts/translationScripts/qsTrId-to-qsTr.py

```diff
@@ -0,0 +1,37 @@
+import fileinput, re, os
```

Comment:
Does it need to be committed? If I understand correctly this is a one off script, and by the look of it, it did his job well